presto-docs/src/main/sphinx/connector/snowflake.rst

```diff
@@ -0,0 +1,111 @@
+with a different name (making sure it ends in ``.properties``). For
+example, if you name the property file ``sales.properties``, Presto
+will create a catalog named ``sales`` using the configured connector.
+
+General Configuration Properties
+---------------------------------
+
+================================================== ==================================================================== ===========
+Property Name                                      Description                                                          Default
+================================================== ==================================================================== ===========
+``user-credential-name``                           Name of the ``extraCredentials`` property whose value is the JDBC
+                                                   driver's user name. See ``extraCredentials`` in `Parameter Reference
+                                                   <https://prestodb.io/docs/current/installation/jdbc.html
+                                                   #parameter-reference>`_.
+
+``password-credential-name``                       Name of the ``extraCredentials`` property whose value is the JDBC
+                                                   driver's user password. See ``extraCredentials`` in `Parameter
+                                                   Reference <https://prestodb.io/docs/current/installation/jdbc.html
+                                                   #parameter-reference>`_.
+
+``case-insensitive-name-matching``                 Match dataset and table names case-insensitively.                    ``false``
+
+``case-insensitive-name-matching.cache-ttl``       Duration for which remote dataset and table names will be
+                                                   cached. Set to ``0ms`` to disable the cache.                         ``1m``
+================================================== ==================================================================== ===========
+
+Querying Snowflake
+--------------
+
+The Snowflake connector provides a schema for every Snowflake *database*.
+You can see the available Snowflake schemas by running ``SHOW SCHEMAS``::
+
+    SHOW SCHEMAS FROM snowflake;
+
+If you have a Snowflake schema named ``web``, you can view the tables
+in this schema by running ``SHOW TABLES``::
+
+    SHOW TABLES FROM snowflake.web;
+
+You can see a list of the columns in the ``clicks`` table in the ``web`` schema
+using either of the following::
+
+    DESCRIBE snowflake.web.clicks;
+    SHOW COLUMNS FROM snowflake.web.clicks;
+
+Finally, you can access the ``clicks`` table in the ``web`` schema::
+
+    SELECT * FROM snowflake.web.clicks;
+
+If you used a different name for your catalog properties file, use
+that catalog name instead of ``snowflake`` in the above examples.
+
+Snowflake Connector Limitations
+---------------------------
+
+The following SQL statements are not yet supported:
```

Comment:
```suggestion
The following SQL statements are not supported:
```
`yet` implies a promise that Presto might support these SQL statements - which is true, but might not or never is also true. Removing `yet` avoids the implication and improves the time-stableness of the documentation: if a user reads `yet`, then checks the last update of this doc page was four years ago, they can feel frustration. 

In general, technical documentation should use the present tense and should describe only the current state of the software. For references, see the Google developer documentation style guide about [present tense](https://developers.google.com/style/tense), [Document future features](https://developers.google.com/style/future), and [Timeless documentation](https://developers.google.com/style/timeless-documentation).